completable-reactor-runtime/src/main/kotlin/ru/fix/completable/reactor/runtime/execution/MergeByExecutionBuilder.kt

```diff
@@ -291,9 +291,11 @@ class MergeByExecutionBuilder<PayloadType>(
                         null
                     }
 
-            val mergeStatus = pvx.invokeMergingMethod(payload, handlingResult)
-
-            mergeCall.stop()
+            val mergeStatus = mergeCall.use {
+                pvx.invokeMergingMethod(payload, handlingResult).also {
+                    mergeCall.stop()
+                }
+            }
```

Comment:
It is not a kotlin way to use variable by its source name if method `use, also, appy, run, ...` used. I suggest:

```suggestion
            val mergeStatus = mergeCall.use { profiledCall ->
                pvx.invokeMergingMethod(payload, handlingResult).also {
                    profiledCall.stop()
                }
            }
```

Reply:
good point for kotlin way )

Reply:
fixed